### PR TITLE
Fix for #627 - Link wrapped images

### PR DESCRIPTION
**Pull request by mensab**

### CODE REVIEW

File: Awful.apk/src/main/java/com/ferg/awfulapp/thread/AwfulPost.java [hunk 1]
```diff
@@ -525,7 +525,20 @@ public static void processPostImage(Element img, boolean isOldImage, AwfulPrefer
                 // switch out for a link with the url
                 img.replaceWith(new Element(Tag.valueOf("a"), "").attr("href", originalUrl).text(originalUrl));
             } else {
-                img.replaceWith(new Element(Tag.valueOf("p"), "").text(originalUrl));
+                if (alreadyLinked) {
+                    Element parent = img.parent();
+
+                    parent.appendText(img.parent().attr("href"));
```
Comment (Sereri) [inline Awful.apk/src/main/java/com/ferg/awfulapp/thread/AwfulPost.java]: `parent.appendText(parent.attr("href"));`

**Comment by baka-kaba** [comment]
We need some CSS styling for this - from a little messing around I got this:

image

with this in ``general.css``
````
.converted-to-link {
    display: block;
    border: 2px #00c0f4 solid;
    padding: 8px;
}

.converted-to-link::before {
	content: "Image converted to link";
	font-style: italic;
	font-size: 0.8em;
}

.converted-to-link a::before {
	padding-right: 2px
}

.converted-to-link a {
	display: block
}

.a-link {
    padding-bottom: 1.5em;
}

.a-link::before {
	content: "🔗";
}

.img-link::before {
	content: "🖼";
}
````

The padding needs adjusting and honestly, I think material icons for the link and image would look way better (and my CSS is bad) but how's that looking? Also for consistency I think it would be a good idea to wrap the converted image too, like this?

image

And I think it might be better to have the image url first, _then_ the ``a`` link - I feel like the image is the "main" element, yknow?

**Comment by mensab** [comment]
I've updated it to put the image url before the a link.

My css skills are non-existent.

**Comment by baka-kaba** [comment]
I just poke at the CSS in the ~wallet inspector~ device inspector in Chrome. Sereri's reworking that stuff at the moment though!

**Comment by baka-kaba** [comment]
Looks good and seems to work!

I tweaked the CSS since the elements are switched around - this should work for plain images that are converted to links too (if we add the classes to those as well)

````
.converted-to-link {
    display: block;
    border: 2px #00c0f4 solid;
    padding: 8px;
}

.converted-to-link::before {
	content: "Image converted to link";
	font-style: italic;
	font-size: 0.8em;
}

.converted-to-link a::before {
	padding-right: 2px
}

.converted-to-link a {
	display: block;
	padding-top: 8px;
	padding-bottom: 8px;
}

.a-link::before {
	content: "🔗";
}

.img-link::before {
	content: "🖼";
}
````

**Pull request merged**